docs/source/tutorial/data-source.md

```diff
@@ -49,7 +49,7 @@ _src/datasources/launch.js_
 ```js
 async getAllLaunches() {
   const response = await this.get('launches');
-  return Array.isArray(result)
+  return Array.isArray(response)
```

Comment:
I think both of these variables actually need to be changed to `res` in order to match up with [the `fullstack-tutorial` repository's](https://github.com/apollographql/fullstack-tutorial) "final" state, as seen in [this commit](https://github.com/apollographql/fullstack-tutorial/blob/7948b85d747300fd4d1596b6173b9ee783431c75/final/server/src/datasources/launch.js#L28-L33).

Reply:
Let's continue with `response` and change the final state of the `fullstack-tutorial`. I think #294 makes a very good point about `res` vs `response`

Reply:
I also agree with, @cdimitroulas!

Let’s make sure we update the tutorial!

Reply:
yay for clearer names :hugs: